config/settings.yml

```diff
@@ -121,3 +121,5 @@ recaptcha:
 derivative:
   # Choose whether collection managers and admins can download high quality derivatives
   allow_download: true
+# Maximum size for uploaded files in bytes (default is disabled)
+max_upload_size: 2147483648 # Use :none or comment out to disable limit
```

Comment:
Should this be `:none` since the comment specifies the default is for the limit to be disabled?

Reply:
I was debating about this since it could be a surprising change for sysadmins but the ticket says disable so I'll make that change and will add a warning in the upgrade notes.